<Issue id="12736" key="FAB-435" number="435" project="10002" reporter="zmanian" creator="zmanian" type="10004" summary="Sending an invoke transaction with confidentiality enabled to a chaincode without it enabled cause the chaincode to exit." priority="3" resolution="10000" status="6" created="2016-09-24 16:45:00.0" updated="2018-07-20 14:10:14.0" resolutiondate="2017-04-11 19:04:36.0" votes="0" watches="4" workflowId="36269"> <description><! CDATA Documenting observed and partially investigated behavior.  1. Launch a fabric network with confidentiality disabled.  2. Deploy chaincode  3. Invoke the chaincode from the SDK with confidentiality enabled.  4. the chaincode process exits with ErrDifferrentConfidentialityProtocolVersion.  What I suspect should have happened is that the invoke tx should never have been submitted to the chaincode process with the confidentiality mismatch.    ></description> </Issue>
